foundationdb/src/main/java/site/ycsb/db/foundationdb/FoundationDBClient.java

```diff
@@ -102,6 +118,7 @@ public void init() throws DBException {
             if (traceDirectory != TRACE_DIRECTORY_DEFAULT) {
               logger.info("FDB trace directory: {}", traceDirectory);
               fdb.options().setTraceEnable(traceDirectory);
+              fdb.options().setTraceFormat(traceFormat);
```

Comment:
Put this line in a if(setTransactionTrace)?

Reply:
The function of this line is set log format which is different from the function of transaction trace. For this line, I simply make ycsb configurable for the format of generated logs. In the code, the format setting is only useful when client logging is open, in the case of traceDirectory != TRACE_DIRECTORY_DEFAULT.